doc/*: add man pages

fluorescent-haze opened the pull request:
- doc/tinyramfs.8: add manpage for tinyramfs
- doc/tinyramfs.5: add configuration manpage
- Makefile: install manpages

illiliti reviewed (changes requested):
Thank you. Looks pretty good overall. I can merge this, but i would prefer if this would be rewritten in scdoc format. What do you think?

fluorescent-haze commented:
Thanks for taking a look. It's my first time writing a man page, so I'm bound to make mistakes. I'll fix the things you pointed out. I wrote it in mdoc, but I'll take a look at scdoc.

fluorescent-haze commented:
I rewrote the man pages in scdoc with your requested changes, which has a very nice syntax compared to plain mdoc. I also added a job in the Makefile to 'make doc' which just runs the doc/*.scd files through scdoc. Lemme know if there's anything else you want me to change.

illiliti commented on doc/tinyramfs.5.scd:
```diff
@@ -0,0 +1,98 @@
+*UUID*, *LABEL*, and *PARTUUID* (eg. LABEL=<label>). This requires a blkid 
+utility which supports that identifier.
+
+root
```
Add additional newline between option and description here. Do same for other options.

fluorescent-haze replied:
Multiple newlines don't show up in the output file, so I'll use '++' so it does, as I assume you'd want.

illiliti commented on doc/tinyramfs.5.scd:
```diff
@@ -0,0 +1,98 @@
+tinyramfs(5) "tinyramfs" "2022-05-09"
+# NAME
```
Add additional newline between title and text here. Do same for others.

illiliti reviewed (changes requested):
Great work! Here's some nitpicks which needs to fixed before i can merge this. Thanks again.

fluorescent-haze commented:
Everything should be good now!
